nginx/etc/nginx/nginx-ssl.conf.ctmpl

```diff
@@ -0,0 +1,102 @@
+worker_processes  1;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+
+    {{ $acme_domain := env "ACME_DOMAIN" }}
+
+    {{ if service "docker-registry" }}
+    upstream docker-registry {
+        # write the address:port pairs for each healthy docker-registry node
+        {{range service "docker-registry"}}
+        server {{.Address}}:{{.Port}};
+        {{end}}
+        least_conn;
+    }{{ end }}
+
+    ## Set a variable to help us decide if we need to add the
+    ## 'Docker-Distribution-Api-Version' header.
+    ## The registry always sets this header.
+    ## In the case of nginx performing auth, the header will be unset
+    ## since nginx is auth-ing before proxying.
+    map $upstream_http_docker_distribution_api_version $docker_distribution_api_version {
+        'registry/2.0' '';
```

Comment:
This seems to set the header to:
```
docker-distribution-api-version: registry/2.0, \
```
which then confuses sdc-docker when checking against this header (sdc-docker uses this to see if the registry supports v2).